src/SFA.DAS.EAS.Employer_Account.Database/Scripts/Manual/CON-3921-DataFixForTermAndConditionsAcceptedOn.sql

```diff
@@ -0,0 +1,11 @@
+﻿-- Update TermsAndConditionsAcceptedOn for all users created afer 27th July 2021
+	 Update [employer_account].[User] 
+     Set TermAndConditionsAcceptedOn = GETDATE()
```

Comment:
can we set this to the CreatedDate. This would be a more accurate reflection of when the terms and conditions were "accepted"